README (2).md

```diff
@@ -0,0 +1,49 @@
+![PollCord_Logo](https://user-images.githubusercontent.com/36363608/193755078-ac3949c0-0db9-4697-bf99-cc62e3788621.png)
+
+
+[![MIT License](https://img.shields.io/badge/License-MIT-green.svg)](https://choosealicense.com/licenses/mit/)
+
+[![Python](https://img.shields.io/badge/python-v3.7+-green.svg)](https://docs.python.org/3/)
```

Comment:
Similar to how ClassMateBot does it, try and have these badges and elements centered instead of left-aligned.